PeerioTechnologies/peerio-desktop
# chore(deps): update deps

#### Relevant info and issue/PR links

s👏e👏m👏v👏e👏r👏

#### Testing instructions

- Avatar upload: check that you can edit avatar and upload it.
- Chat message formatting (specifically urls should be still linkified): check that messages are displayed and URLs are still linkified.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
